Assets/Scripts/Game/MagicAndEffects/EntityEffectManager.cs

```diff
@@ -316,8 +318,12 @@ public bool SetReadySpell(EntityEffectBundle spell, bool noSpellPointCost = fals
             if (spell == null || spell.Settings.Version < minAcceptedSpellVersion)
                 return false;
 
+            //By default, enemy spell costs are calculated using player-character skill levels.
```

Comment:
Is this confirmed to be an original DF bug? I'd rather we just fix that. For what it's worth, we've fixed it for Dodge, and this would be a lot less controversial

Reply:
I brought this up on dfworkshop a while back.
See:
 https://forums.dfworkshop.net/viewtopic.php?p=64057&hilit=spell+cost#p64057